app/models.py

```diff
@@ -1297,6 +1297,7 @@ def check_code(self, cde):
     NOTIFICATION_SENDING,
     NOTIFICATION_SENT,
     NOTIFICATION_DELIVERED,
+    NOTIFICATION_PENDING,
```

Comment:
@leohemsted Based on your context, you are still fine with this being added to `NOTIFICATION_STATUS_TYPES_BILLABLE`, rather than us creating a new `NOTIFICATION_STATUS_TYPES_BILLABLE_FOR_SMS`?

Reply:
I'm ok with that. It's not stated anywhere in the code, but we do only use it in the Firetext callback scenario.